api.js

```diff
@@ -195,7 +195,7 @@ function handlePaths(files) {
 		files = [
 			'test.js',
 			'test-*.js',
-			'test/*.js'
```

Comment:
@jamestalmage: yes, it would be a breaking change. But otherwise we have Inconsistent and confusing behavior.